compiler/damlc/tests/BUILD.bazel

```diff
@@ -702,7 +702,6 @@ da_haskell_test(
         ":repl-test.dar",
         "//compiler/damlc",
         "//daml-script/daml:daml-script.dar",
-        "//test-common/canton:lib",
```

Comment:
What's not used ?

Reply:
No it wasn't! The Canton sandbox is instantiated via a function provided by `//libs-haskell/test-utils`, which packs the jar already.

Reply:
:+1: